MovieSharing/VideoManager.swift

```diff
@@ -9,20 +9,30 @@ import Foundation
 
 class VideoManager{
     
+    // MARK: - Types
+    //FIX ME:
+    enum VideoError: Error {
```

Comment:
- [ ] you'll adjust/elaborte this anyway in another ticket but even for now it would be good to keep the error types consistent, therefore I would suggest moving the Error enum to a seperate file, rename it to something like `NetworkingError` and then use that in the fetchVideo completion as well